### DISCUSSION TITLE
Pinot Select query issue

### DISCUSSION THREAD

**Discussion by Asha245** [trinodb/trino#19267] [Q&A]
Apart from select query all other queries are working fine.

presto:default> select * from transcript;
Query 20231004_164227_00051_ji9z8 failed: java.net.NoRouteToHostException: No route to host

**Comment by lucas-0liveira** [comment]
It seems like you're facing an issue while executing the query in the database using Presto. The error indicates there is a connection problem with the specified host.

The error "java.net.NoRouteToHostException: No route to host" typically occurs when the system can't find a route to communicate with the specified host. This can be caused by various reasons such as network issues, firewall settings, or the host not being available at the moment.
Hope I've been helpful.
